src/poetry/core/spdx/license.py

```diff
@@ -3,6 +3,10 @@
 
 
 class License(namedtuple("License", "id name is_osi_approved is_deprecated")):
+    id: str
+    name: str
+    is_osi_approved: bool
+    is_deprecated: bool
```

Comment:
What do you think about using [typing.NamedTuple](https://docs.python.org/3/library/typing.html#typing.NamedTuple) here?

Reply:
No view!  Probably would be fine.

Reply:
Would be fine if `typing.NamedTuple` will then be used.